accepted/future-releases/type-modifiers/feature-specification.md

```diff
@@ -1,11 +1,13 @@
-# "Base", "interface", and "final" types
+# Type modifiers
```

Comment:
Mmm.  I'd really prefer we not use this terminology.  Classes are not types.  Classes *produce* types (among other things).  That is, a class declaration `class C` introduces a type named `C`, and it also introduces a bundle of code, with a namespace, some semantics, etc.  When we say `class C extends D`, `D` here is not the *type* `D`, it is the bundle of stuff that we also happen to name (in the surface syntax) `D`.   "Type modifiers" suggest things that I can use to modify types (e.g. `sealed int x = 3`), which here declares a variable `x` of type `sealed x`, whatever that might be.  Perhaps "declaration modifiers"?  Or just stick with "class"?

Reply:
I'm a little worried that it might be confusing that the proposal applies to mixins too if we call it "class modifiers". "Declaration modifiers" is a little vague: I think of variables and methods as declarations too and I specifically didn't call this proposal "capability modifiers" to scope it to just type declarations and not other kinds of declarations.

"Class and mixin modifiers" is a mouthful. I wanted something relative short so we can also use it for the experiment flag. "Class modifiers" is probably the least bad.

@kallentu @bwilkerson, thoughts on the name?

Reply:
I can live with "class modifiers", since mixins are only used to build classes anyway,.

Reply:
I originally called them 'Class modifiers' so that works for me.